tests/node.rs

```diff
@@ -20,7 +20,7 @@ mod node {
     fn set_measure() {
         let mut taffy = Taffy::new();
         let node = taffy
-            .new_leaf(FlexboxLayout::default(), MeasureFunc::Raw(|_| Size { width: 200.0, height: 200.0 }))
+            .new_leaf_with_measure(FlexboxLayout::default(), MeasureFunc::Raw(|_| Size { width: 200.0, height: 200.0 }))
```

Comment:
This can be migrated to the new method.

Reply:
It was this, then two other occurrences of this; however those were specifically testing the `MeasureFunc` so migrating those does not make sense.

Reply:
Update: This test is specifically testing the behaviour of first setting one `MeasureFunc,` then setting a different, hence this should not be changed.

Reply:
Alright, sounds good.